specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -1842,6 +1842,22 @@
         "sourceVault"
       ],
       "description": "Key Vault Key Url and vault id of KeK, KeK is optional and when provided is used to unwrap the encryptionKey"
+    },
+	"KeyForDiskEncryptionSet": {
+      "properties": {
+        "sourceVault": {
+          "$ref": "#/definitions/SourceVault",
+          "description": "Resource id of the KeyVault containing the key or secret. This property is optional and cannot be used if the KeyVault subscription is not the same as the Disk Encryption Set subscription."
```

Comment:
@vikramd-ms - Not sure if you were looking at the correct description on your comment.
Is this description clear or does it need to be updated ?